src/auslib/AUS.py

```diff
@@ -34,14 +35,29 @@ def isSpecialURL(url, specialForceHosts):
 def isForbiddenUrl(url, product, allowlistedDomains):
     if allowlistedDomains is None:
         allowlistedDomains = []
-    domain = urlparse(url)[1]
+    parsedUrl = urlparse(url)
+    domain = parsedUrl[1]
```

Comment:
I realize that you're inheriting this, but it would be a good time to update this to use [the named attributes that urlparse returns](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse) (it looks like you want `.netloc` and `.path` specifically).

Reply:
Sure, I noticed that from the docs, but chose to follow the convention set in the code. Fixed!